src/transactions.ts

```diff
@@ -174,6 +183,19 @@ export function actionFromRLP(sdk: SDK, rlp: Buffer): Action {
                 deposit: decodeU64(decoded[1]),
                 metadata: decoded[2]
             };
+        case CHANGE_PARAMS_ACTION_ID:
+            if (decoded.length <= 3) {
+                throw new Error(
+                    "A length of a RLP list of a changeParams action should be more than 3"
```

Comment:
```suggestion
                    "A length of a RLP list of a changeParams action should be more than 4"
```

Reply:
Is it right to say "should be more than 3."
"x should be more than 3" == (x > 3)
"x should be more than 4" == (x > 4)

Reply:
Sorry! You're right. It was my mistake.